doc/design/staging-1.md

```diff
@@ -24,7 +24,7 @@ The staging server will track the `master` branch in `dandi-api` and will automa
 
 The production server will not track any branch. Instead, it will automatically redeploy whenever a release is tagged in GitHub.
 
-The staging web GUI will track the `master` branch in `dandiarchive` and will automatically redeploy whenever `master` is changed.
+The staging web GUI will track the `staging` branch in `dandiarchive` and automatically redeploy when the `staging` branch is changed. The `staging` branch will stay "synced" with `master` via a GitHub Action that will rebase `staging` on `master` whenever a change is pushed to `master`.
```

Comment:
Why bother rebasing? Wouldn't it likely cause issues for PRs based on saying prior state of staging?

Reply:
PRs should never be based off of the `staging` branch. In our current staging setup, the `staging` branch is basically a "read-only" mirror of `master` except it uses `api-staging.dandiarchive.org` as the server instead of `api.dandiarchive.org`.

Reply:
The `staging` branch is always the same as `master`, with just the changes required to point the deployment at the staging API rather than the production API. Therefore, no PR should ever be targeted to `staging`, and rebasing this way is the appropriate way to set things up for the staging deploy.